pandas/core/strings/accessor.py

```diff
@@ -2482,12 +2482,16 @@ def get_dummies(
         1   False  False   False
         2   True   False   True
         """
+        from pandas.core.dtypes.common import is_string_dtype
```

Comment:
`pandas.core.dtypes.common` is already being imported from at the top of this file. Can you move this import up there.